tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -184,11 +184,9 @@ def create_parser(product_choices=None):
     debugging_group.add_argument("--repeat-until-unexpected", action="store_true", default=None,
                                  help="Run tests in a loop until one returns an unexpected result")
     debugging_group.add_argument('--retry-unexpected', type=int, default=0,
-                                 help=('Maximum number of times to retry '
-                                       'each test that consistently runs '
-                                       'unexpectedly in the initial repeat '
-                                       'loop. A retried test takes any '
-                                       'expected status as its final result.'))
+                                 help=('Maximum number of times to retry unexpectedly failed tests.'
+                                       'A test is retried until it gets one of the expected status or pass,'
```

Comment:
Should unexpected passes remove a test from the retry pool even without `--no-fail-on-unexpected-pass`? I ask because other CI systems (notably, this repo) will block changes with unexpected passes; Chromium does not.

Unfortunately, I didn't consider this when writing [the RFC](https://github.com/web-platform-tests/wpt/pull/35925).

Reply:
Right. I think this is a good point. When run without "--no-fail-on-unexpected-pass", unexpected pass tests should be retried. Will make a change.

Reply:
Done